src/__tests__/integration/index.js

```diff
@@ -0,0 +1,44 @@
+// Jest trying to start all files under /__tests__ directory
```

Comment:
Instead of fake test I have modified test run command in `package.json` using the following parameter:
```
--testMatch '**/__tests__/**/*.test.js'
```

Also added test URL modification as was described in a separate comment.

Note that when running in WebStorm you might need to add those params in its run configuration:
![image](https://user-images.githubusercontent.com/5430498/77823081-a1b72480-7109-11ea-8001-5af008764352.png)